src/components/src/main/java/org/apache/jmeter/extractor/BoundaryExtractor.java

```diff
@@ -243,9 +243,23 @@ private String getInputString(SampleResult result) {
         if (StringUtils.isBlank(inputString)) {
             return Collections.emptyList();
         }
-        Objects.requireNonNull(leftBoundary);
-        Objects.requireNonNull(rightBoundary);
-
+		boolean isEmptyLeftBoundary = StringUtils.isEmpty(leftBoundary);
```

Comment:
Please use 4 spaces not tabs for indents.

Reply:
@ham1 Thanks, fixed indentations with 4 spaces